Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles.Tests/ErrorTests.cs

```diff
@@ -59,7 +59,6 @@ public void DownloadFiles_TestWithSubDirNameAsFileMask()
 
             var ex = Assert.Throws<Exception>(() => SFTP.DownloadFiles(source, _destination, _connection, _options, _info, new CancellationToken()));
             Assert.IsTrue(ex.Message.StartsWith("SFTP transfer failed: 1 Errors: No source files found from directory"));
-
             Helpers.DeleteSubDirectory(path);
         }
 
```

Comment:
https://github.com/FrendsPlatform/Frends.SFTP/blob/267e98ab232f82b7a9f5fa1cee119a75ec7687d5/Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles.Tests/ErrorTests.cs#L80
Please use `Assert.IsTrue` instead of `That`.

https://github.com/FrendsPlatform/Frends.SFTP/blob/267e98ab232f82b7a9f5fa1cee119a75ec7687d5/Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles.Tests/ErrorTests.cs#L119
Please use `Assert.IsTrue` instead of `That`.

https://github.com/FrendsPlatform/Frends.SFTP/blob/267e98ab232f82b7a9f5fa1cee119a75ec7687d5/Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles.Tests/ErrorTests.cs#L148
Please use `Assert.IsTrue` instead of `That`.